README.md

```diff
@@ -84,6 +84,26 @@ The output should look like this:
 }
 ```
 
+### Cloud Spanner
+We use Cloud Spanner to store the ELO scores, per model, for each rating done. Before running the application. 
+
```

Comment:
Can we mention that due to the indexes on model, time, etc (whichever the indexes are on) allows for interesting queries like ... elo score over time, etc.